doc/config.rst

```diff
@@ -828,6 +828,10 @@ that use Sphinx's HTMLWriter class.
      between the ``'sourcelink.html'`` and ``'searchbox.html'`` entries.  This
      is for compatibility with Sphinx versions before 1.0.
 
+   .. deprecated:: 2.0
+
+      a single string value for ``html_sidebars`` will be deprecated in 2.0
+
```

Comment:
It is not clear to me if you mean it is deprecated at 1.7 and removed at 2.0 or simply deprecated at 2.0.